src/test/kotlin/com/example/passmanager/repositories/PassOwnerRepositoryImplTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.example.passmanager.repositories
+
+import com.example.passmanager.domain.MongoPassOwner
+import com.example.passmanager.testcontainers.WithMongoTestContainer
+import com.example.passmanager.util.OptimisticLockTestUtils.getOptimisticLocksAmount
+import com.example.passmanager.util.PassOwnerFixture.getOwnerWithUniqueFields
+import com.example.passmanager.util.PassOwnerFixture.passOwnerToCreate
+import org.assertj.core.api.Assertions.assertThat
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.exists
+import org.springframework.data.mongodb.core.findById
+import org.springframework.data.mongodb.core.query.Criteria.where
+import org.springframework.data.mongodb.core.query.Query.query
+import org.springframework.data.mongodb.core.query.isEqualTo
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+
+@SpringBootTest
```

Comment:
How many Spring contexts will be created during full test run?

Reply:
One, by default, once loaded, the configured ApplicationContext is reused for each test